src/api/api.ts

```diff
@@ -0,0 +1,35 @@
+export interface imageApiModel {
+    imageURL:string;
+    camera:string;
+    rover:string;
+    landingDate:string;
+    launchDate:string;
+    earthDate:string;
+}
+
+export async function imageAPI (date:string,rover:string,camera:string) {
```

Comment:
Can we rename this to 'getImagesFromApi' so it is clear what the function does?
Can we remove async here as we are using fetch?
And can we add spaces between the function parameters?